modules/install-sno-requirements-for-installing-on-a-single-node.adoc

```diff
@@ -10,17 +10,22 @@ Installing {product-title} on a single node alleviates some of the requirements
 
 * *Administration host:* You must have a computer to prepare the ISO, to create the USB boot drive, and to monitor the installation.
 
-* *CPU Architecture:* Installing {product-title} on a single node supports `x86_64` and `arm64` CPU architectures.
+[NOTE]
+====
+ISO is not required for {ibmzProductName} installations.
+====
+
+* *CPU Architecture:* Installing {product-title} on a single node supports `x86_64`, `arm64`,`ppc64le`, and `s390x` CPU architectures.
 
-* *Supported platforms:* Installing {product-title} on a single node is supported on bare metal, vSphere, AWS, Red Hat OpenStack, and {rh-virtualization-first} platforms.
+* *Supported platforms:* Installing {product-title} on a single node is supported on bare metal, vSphere, AWS, Red Hat OpenStack, {rh-virtualization-first}, {ibmpowerProductName}, and{ibmzProductName} platforms.
```

Comment:
@emarcusRH @apinnick  {rh-virtualization-first} didn't resolve in my preview and I assume the entry got missed when you were working on this effort to remove RHV docs: https://issues.redhat.com/browse/OCPRHV-773

Can I simply remove the entry in my PR? Or change to hardcoded?

Reply:
This looks like an error. AFAIK, we have never supported RHV on SNO. I think that should be `{VirtProductName}`.